src/main/java/gov/usgs/cida/pubs/busservice/sipp/SippProcess.java

```diff
@@ -49,24 +48,30 @@ public SippProcess(
 	}
 
 	@Transactional(propagation=Propagation.REQUIRES_NEW)
-	public ProcessSummary processInformationProduct(ProcessType processType, String ipNumber) {
-		ProcessSummary rtn = new ProcessSummary();
-		StringBuilder processingDetails = new StringBuilder(ipNumber + ":");
+	public MpPublication processInformationProduct(ProcessType processType, String ipNumber) {
+		MpPublication rtn = null;
 
+		MpPublication mpPublication = null;
 		InformationProduct informationProduct = getInformationProduct(ipNumber);
-		MpPublication mpPublication = getMpPublication(informationProduct);
+
+		if(informationProduct != null) {
+			mpPublication = getMpPublication(informationProduct);
+		}
+
+		if(mpPublication == null) {
+			mpPublication = new MpPublication();
```

Comment:
However, this instantiation of an MpPublication is not - code in okToProcess (line 173) expects a null mpPublication if none is found on line 58. Either move this if into the else block below (line 68-73), or remove it and instantiate the MpPublication in line 71 (rtn = ...)